src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoryReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'SET_STATUS':
+      return {
+        ...state,
+        status: 'Under construction',
+      };
+    default:
+      return state;
+  }
+};
+
+export default categoryReducer;
```

Comment:
- [ ] Please make sure to create and export action types for checking the status as requested in the project requirements.